crates/sncast/src/starknet_commands/script/mod.rs

```diff
@@ -0,0 +1,23 @@
+use crate::starknet_commands::script::init::Init;
+use clap::{Args, Subcommand};
+
+pub mod init;
+pub mod run;
+
+#[derive(Args)]
+pub struct Script {
+    /// Module name that contains the `main` function, which will be executed
+    pub module_name: Option<String>,
+
+    /// Specifies scarb package to be used
+    #[clap(long)]
+    pub package: Option<String>,
```

Comment:
Since we've added packages support, it now may make sense to use it for init as well (if passed) right?
we could create multiple script packages inside scripts directory + now we probably also should make sure this directory is included in a workspace (since we now support them). Let's try to assess, we might want to consult scarb team on how we should approach this (maybe scarb has a mechanism that fits our needs?)

Reply:
The initial plan was for #1320 to follow immediately after this one, so there shouldn't be any problems because the `package` flag will be used solely for the `run` subcommand.

We can change the order and modify the #1320 allowing it to be merged prior to this PR.

Reply:
as you prefer, it does not make much difference to me tbh

Reply:
Let's stick to the current order then.